translations/base-nl.yaml

```diff
@@ -30,47 +30,47 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.io is een spel over het bouwen van fabrieken, welke steeds moeilijker wordende vormen maken en verfijnen, over een oneindig grote map.
+        Nadat u de gevraagde vormen heeft afgeleverd, komt u verder in de game, en kunt u upgrades verkrijgen om zo uw fabriek te versnellen.
 
-        As the demand for shapes increases, you will have to scale up your factory to meet the demand - Don't forget about resources though, you will have to expand across the [b]infinite map[/b]!
+        Omdat de vraag naar vormen groter wordt, moet u uw fabriek steeds groter maken om aan de vraag te voldoen - vergeet de grondstoffen niet, u zal moeten groeien over de [b]oneindige map[/b]!
+        
+        Binnenkort moet u kleuren mengen en uw vormen ermee verven - Combineer rode, groene en blauwe kleurbronnen om verschillende kleuren, en verf vormen ermee om aan de vraag te voldoen.
 
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        Dit spel heeft 18 opeenvolgende niveaus (Welke u bezig moeten houden voor al gauw een paar uur!) maar ik vroeg constant inhoud toe - er is veel gepland!
 
-        This game features 18 progressive levels (Which should keep you busy for hours already!) but I'm constantly adding new content - There is a lot planned!
+        Door dit spel te kopen krijgt u toegang tot de losstaande versie, welke meer mogelijkheden heeft, mede als toegang gevend aan nieuw toegevoegde onderdelen.
 
-        Purchasing the game gives you access to the standalone version which has additional features and you'll also receive access to newly developed features.
-
-        [b]Standalone Advantages[/b]
+        [b]Losstaande voordelen[/b]
```

Comment:
In the rest of the translation, "standalone" doesn't get translated, except here